Sources/PaymentsCore/Networking/GraphQL/Common/GraphQLClient.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  GraphQLClient.swift
+//  Card
+//
+//  Created by Andres Pelaez on 19/05/22.
+//
+
+import Foundation
+import PaymentsCore
+
+internal class GraphQLClient {
```

Comment:
i think we dont need the `internal` access modifier, as it is the default in swift